kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSession.scala

```diff
@@ -43,7 +43,8 @@ class KyuubiBatchSession(
     className: String,
     batchConf: Map[String, String],
```

Comment:
@turboFei Seems there is no diff between `conf` and `batchConf`, can we merge these two vars into one?

Reply:
merged